CONTRIBUTING.md

```diff
@@ -19,40 +19,48 @@ If you are hired for an Upwork job and have any job-specific questions, please a
+This is the most common scenario for contributors. The Expensify team posts Expensify.cash jobs to the Upwork job list [here](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2). Each job in Upwork has a corresponding GitHub issue, which will include instructions to follow.
 
 #### Proposing a job that Expensify hasn’t posted
 
 In this scenario, it’s possible that you found a bug or enhancement that we haven’t posted to the [Upwork job list](https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&sort=recency&user_location_match=2) or [Github repository](https://github.com/Expensify/Expensify.cash/issues?q=is%3Aissue). This is an opportunity to propose a job, and (optionally) a solution. If it's a valid job proposal, we will compensate you for the solution and give an additional bonus of $150 for proactively proposing the job. In this case, please take the following steps:
 
-    1. Check to ensure an issue does not already exist in the Expensify.cash Issue list or Upwork job list. Please use your best judgement to search for similar titles and issue descriptions. 
-    2. If your bug or enhancement matches an existing issue, please feel free to comment on that GitHub issue with your findings if you think it’ll help solve a problem. 
-    3. If there is no existing issue or Upwork job, create a new GitHub issue in the Expensify.cash repo. 
-    4. Make sure to fill out all the required information fields in the issue template. 
-    5. Optional: If you would like to solve the bug or enhancement that you are proposing, please add a comment on your issue with a solution proposal. 
+    1. Check to ensure an issue does not already exist in the Expensify.cash Issue list or Upwork job list. Please use your best judgement to search for similar titles and issue descriptions.
+    2. If your bug or enhancement matches an existing issue, please feel free to comment on that GitHub issue with your findings if you think it’ll help solve a problem.
+    3. If there is no existing issue or Upwork job, create a new GitHub issue in the Expensify.cash repo.
+    4. Make sure to fill out all the required information fields in the issue template.
+    5. Optional: If you would like to solve the bug or enhancement that you are proposing, please add a comment on your issue with a solution proposal.
     6. Pause on this step until a member of the Expensify team responds on your issue with next steps.
 
+#### Solving Problems
+
+Our problem solving approach at Expensify is to focus on high value problems and avoid small optimizations with results that are difficult to measure. We also prefer to identify and solve problems at their root. Given that, please ensure all proposals fix a specific problem in a measurable way with evidence so they are easy to evaluate. Here's an example of a good problem/solution:
```

Comment:
I kinda feel like "Solving Problems" is not the best title for this section. To me, that title suggests the section would explain "how to solve a problem well."

Since this section is mainly focused on "how Expensify & contributors can submit the most useful proposal", I feel it would be better to title this section "Proposing solutions to problems" or "How to write a great proposal"

Thoughts?

Reply:
Thanks for the suggestion. To be honest,  I think that neither "Proposing solutions to problems" or "How to write a great proposal" encapsulate what I'm trying to communicate here. Making a good _proposal_ to a problem is already defined in the sections below. And people should read this point _after_ they read the "Proposing a job that Expensify hasn’t posted" section.

It's really the lack of _defining the problem_ that people seem to struggle with before they get to know us which is directly tied to the previous section - so maybe this blurb is more of a "note" and should be contained inside the previous section.

Reply:
Ooh I like this - it's not a new section, it's adding additional comments and providing a useful example :)